packages/clerk-js/src/ui/customizables/parseVariables.ts

```diff
@@ -17,10 +17,13 @@ export const createColorScales = (theme: Theme) => {
   const textSecondary = toHSLA(variables.colorTextSecondary) || colors.makeTransparent(variables.colorText, 0.15);
   const textTertiary = toHSLA(variables.colorTextTertiary) || colors.makeTransparent(textSecondary, 0.4);
 
+  const secondaryScale = colorOptionToHslaLightnessScale(variables.colorSecondary, 'secondary');
+
   return removeUndefinedProps({
     ...colorOptionToHslaLightnessScale(variables.colorPrimary, 'primary'),
     ...colorOptionToHslaAlphaScale(variables.colorPrimary, 'primaryAlpha'),
-    ...colorOptionToHslaLightnessScale(variables.colorSecondary, 'secondary'),
+    ...secondaryScale,
+    colorTextSecondary: textSecondary,
```

Comment:
Shouldn't we also pass `colorTextLabel` here?

Reply:
Yeah I missed that, thanks